docker/seahorn/Dockerfile

```diff
@@ -1,16 +1,20 @@
 FROM rvt_yices:latest
 
 USER root
+
+RUN pip3 install pyyaml
+
 RUN mkdir /home/seahorn
 WORKDIR /home/seahorn
 
 ARG VERIFY_COMMON_VERSION
 ARG SEAHORN_VERSION
 ARG YICES_VERSION
 
+# cargo-verify relies on this variable to find the yaml files
 ENV VERIFY_COMMON_DIR=/home/seahorn/verify-c-common
 
-RUN git clone --no-checkout https://github.com/yvizel/verify-c-common.git ${VERIFY_COMMON_DIR} \
+RUN git clone --no-checkout https://github.com/seahorn/verify-c-common.git ${VERIFY_COMMON_DIR} \
   && cd ${VERIFY_COMMON_DIR} \
   && git checkout ${VERIFY_COMMON_VERSION}
```

Comment:
Can you change the name of this environment variable to include the word "SEAHORN"?
Ditto for adding the command line argument to cargo-verify.

As it is, the directory does not look very "common" since it is SeaHorn specific and it's not obvious how we would generalize it for KLEE, SMACK, Crux or any other tool that we want to pass configuration files to.

Reply:
Done